docs/viplab-3.0/computation_task.md

```diff
@@ -55,6 +55,348 @@ the following structure:
+            "selected" : true
+          },
+          {
+            "value" : "All kinds of Bugs (also the ones living in your Computer)"
+          },
+          {
+            "value" : "I never dislike anything!"
+          }
+        ], 
+        "validation": "anyof"
+      }
+      </pre>
+      Toggle Result:
+      <pre>
+      ["Spiders"]
+      </pre>
+    </td>
+    <td>
+      Array of Strings
+    </td>
+  </tr>
+  <tr>
+    <td>slider (single)</td>
+    <td>
+      Slider (single) Example:
+      <pre>
+      {
+        "mode" : "any",
+        "identifier" : "__sliderSingle__", 
+        "metadata" : {
+          "guiType" : "slider",
+          "name": "Temperature",
+          "vertical": false,
+          "decription" : "How hot do you like your coffee? (in degrees Celsius)"
+        },
+        "default": [
+          75
+        ],
+        "min": 0,
+        "max": 90,
+        "step": 10,
+        "validation": "range"
+      }
+      </pre>
+      Slider (single) Result:
+      <pre>
+      75
+      </pre>
+    </td>
+    <td>
+      Number
+    </td>
+  </tr>
+  <tr>
+    <td>slider (multiple)</td>
+    <td>
+      Slider (multiple) Example:
+      <pre>
+      {
+        "mode" : "any",
+        "identifier" : "__sliderMultiple__", 
+        "metadata" : {
+          "guiType" : "slider",
+          "name": "random numbers",
+          "vertical": true,
+          "decription" : "Choose three random numbers to be output by the container"
+        },
+        "default": [
+          25,
+          50,
+          75
+        ],
+        "min": 0,
+        "max": 100,
+        "step": 5,
+        "validation": "range"
+      }
+      </pre>
+      Slider (multiple) Result:
+      <pre>
+      [25, 50, 75]
+      </pre>
+    </td>
+    <td>
+      Array of Numbers
+    </td>
+  </tr>
+  <tr>
+    <td>input_field</td>
+    <td>
+      Input_field Example:
+      <pre>
+      {
+        "mode" : "any",
+        "identifier" : "__inputText__", 
+        "metadata" : {
+          "guiType" : "input_field",
+          "type": "text",
+          "name": "name",
+          "decription" : "Enter your name"
+        },
+        "default" : [""],
+        "validation": "none"
+      }
+      </pre>
+      Input_field Result:
+      <pre>
+      "base64:S2F0aHJ5bg"
+      </pre>
+    </td>
+    <td>
+      If the input_field is of type number: Number
+      Else: Prefixed ("base64:") and Base64URL-encoded String - is decoded by Websocket API, before it is sent to Backend
+    </td>
+  </tr>
+  <tr>
+    <td>editor</td>
+    <td>
+      Editor Example:
+      <pre>
+      {
+        "mode" : "any",
+        "identifier" : "__default__", 
+        "metadata" : {
+          "guiType" : "editor", 
+          "name": "code",
+          "decription" : "Enter some code"
+        },
+        "default": ["aW50IG1haW4oaW50IGFyZ2MsIGNoYXIgKiphcmd2KSB7IC8vIFByaW50ICdIZWxsbyBXb3JsZCcgfQ"],
+        "validation": "pattern"
```

Comment:
The pattern validator requires a pattern to be given. Maybe there should be no validator for this editor